docs/GetStarted/buildRPC.md

```diff
@@ -12,66 +20,101 @@ After running the RPC, the Intel® AMT managed device can be managed remotely us
+         4. Follow the remaining instructions. 
 
-6. Type "v" to see the list of release branches from the **Use workflow from** dropdown.
+    === "Windows"
+         **To install prerequisites on Windows:**
 
-7. Select the v{{ rpcClone.version }} branch.
+         1. See Go's [Download and Install](https://golang.org/doc/install).
+         2. Choose and download a distribution appropriate for your managed device and operating system (e.g., msi).
+         3. Run the downloaded file and follow prompts to install. 
+         4. See [tdm-gcc](https://jmeubank.github.io/tdm-gcc/).
+         5. Choose a version and download the .exe. 
+         6. Run the downloaded file and follow prompts to install. For a new installation, choose **Create** and accept all default installation options.
 
-8. By default, the Build Type should be **release**.  
 
-9. Click the **Run Workflow** button.
+**To verify Go and tdm-gcc installations:**
 
-10. The build time ranges from 15 to 20 minutes.
+1. Open a Terminal or Command Prompt: 
+   ``` bash
+   go version
+   ```
+   For Windows only: 
+   ``` bash
+   gcc -v
+   ```
+2. Confirm the version numbers.
 
-11. Once the download is complete, click the completed job which will feature a green checkmark.
+## Get the RPC
 
-12. Download the appropriate RPC for your managed device's OS under the **Artifacts** section.
+**To clone the repository:**
 
+1. Open a Terminal or Command Prompt and navigate to a directory of your choice for development:
+   ``` bash
+   git clone --recursive https://github.com/open-amt-cloud-toolkit/rpc-go
+   ```
+  
+2. Change to the cloned `rpc-go` directory:
+   ``` bash
+   cd rpc-go
+   ```
 
-### Docker Build  (For Linux Hosts)
+## Build the RPC
 
-Keep in mind, the image created with this method is only suitable for Docker on a Linux host.
+**To build the executable:**
 
-To build RPC with Docker, use the following command from the open-amt-cloud-toolkit/rpc directory:
+1. Open a Terminal (Linux) or Powershell/Command Prompt **as Administrator** (Windows):
 
-``` bash
-docker build -f "Dockerfile" -t rpc:latest .
-```  
+    === "Linux"
+        ``` bash
+        sudo apt install build-essential
+        ```
+        ``` bash
+        go build -o rpc ./cmd
+        ```
+    === "Docker (On Linux Host Only)"
+        ``` bash
+        docker build -f "Dockerfile" -t rpc-go:latest .
+        ```
+    === "Windows"
+        ``` bash
+        go build -o rpc.exe ./cmd
+        ```
+    !!! note
+        The image created with the Docker instruction above is only suitable for Docker on a Linux host.
 
-!!! note
-    If you wish to perform this docker build on Windows, ensure **Use the WSL 2 based engine** is enabled in Docker Desktop.
-    Once the build is complete, you will need to get the built image to your Linux system running Docker.
 
+2. Confirm a successful build:
 
+    === "Linux"
+        ``` bash
+        sudo rpc version
```

Comment:
Double check all the Linux cmds, they need to be 'sudo ./rpc cmd-here', ./ is requried